app/src/main/java/com/swent/skillswap/ui/profile/SkillsEditScreen.kt

```diff
@@ -0,0 +1,215 @@
+                        value = query,
+                        onValueChange = { query = it },
+                        label = { Text("Search skills", color = Color.White.copy(alpha = 0.7f)) },
+                        placeholder = {
+                            Text("Type to search...", color = Color.White.copy(alpha = 0.5f))
+                        },
+                        modifier =
+                            Modifier.menuAnchor()
+                                .onFocusChanged { hasFocus = it.isFocused }
+                                .fillMaxWidth(),
+                        colors =
+                            TextFieldDefaults.colors(
+                                focusedTextColor = Color.White,
+                                unfocusedTextColor = Color.White,
+                                focusedIndicatorColor = Color.Transparent,
+                                unfocusedIndicatorColor = Color.Transparent,
+                                disabledIndicatorColor = Color.Transparent
+                            ),
+                        shape = RoundedCornerShape(8.dp)
+                    )
+
+                    DropdownMenu(
+                        expanded = expanded && hasFocus && suggestions.isNotEmpty(),
+                        onDismissRequest = { expanded = false },
+                        properties = PopupProperties(focusable = false),
+                        modifier = Modifier.fillMaxWidth(0.8f)
+                    ) {
+                        suggestions.forEach { skill ->
+                            DropdownMenuItem(
+                                text = {
+                                    Text(
+                                        skill.name.replace("_", " ").lowercase().replaceFirstChar {
+                                            if (it.isLowerCase()) it.titlecase() else it.toString()
+                                        },
+                                        color = Color.Black
+                                    )
+                                },
+                                onClick = {
+                                    selectedSkills = selectedSkills + skill
+                                    query = ""
+                                    expanded = false
+                                }
+                            )
+                        }
+                    }
+                }
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                // Selected skills display
+                Text(
+                    text = "Selected Skills (${selectedSkills.size}):",
+                    fontSize = 14.sp,
+                    color = Color.White.copy(alpha = 0.8f),
+                    modifier = Modifier.padding(bottom = 8.dp)
+                )
+
+                Box(modifier = Modifier.height(120.dp).fillMaxWidth()) {
+                    val flowScroll = rememberScrollState()
+                    FlowRow(
```

Comment:
FlowRow is an experimental feature: if you want to use it (which I wuldn't recommend for a stable apk) write this on line 34

*@OptIn(ExperimentalMaterial3Api::class, ExperimentalLayoutApi::class)*